### ISSUE TITLE
Mac app looks like Windows UI in 1.5.16 update on MacOS 10.13.6

### ISSUE THREAD

**Issue by daniel-toman** [crystalidea/macs-fan-control#771]
Hello, I've been using MFC for over 5 years and love it! Recently when I updated to the latest version 1.5.16, it seems that the UI has changed to kind of look like a Windows or Linux app? It looked like a native Mac app before. This is on an old 2010 Mac Mini running High Sierra. It looks like I have the same version of MFC installed on my newer 2019 Macbook Pro running Sonoma, and on that system MFC looks like a normal Mac app.

I updated from within the app when prompted by the app's auto updater, I was on a fairly recent version before this, I think it was a pretty minor version number update.

I've tried quitting the app and relaunching, I've also tried restarting the computer, and it still has the same WIndows-y look.

Screenshots and tech info zip file attached.

[Macmini4,1_tech_info.zip](https://github.com/user-attachments/files/16769619/Macmini4.1_tech_info.zip)
![Screen Shot 2024-08-27 at 1 57 30 PM](https://github.com/user-attachments/assets/b3b95dc0-5898-49f1-9756-457fea4ac1e2)

<details>
Mac Mini 2010 on MacOS 10.13.6 - UI looks like Windows.

Macs Fan Control, Pro license, just 1.5.16 upgraded from a recent prior version.

Macbook Pro 2019 on macOS 14.13.1 - UI looks like Mac.
</details>

**Comment by disnos9** [comment]
downgrade then

**Comment by kleuter** [comment]
Have you tried 1.5.15?
https://github.com/crystalidea/macs-fan-control/releases

**Comment by kleuter** [comment]
Please post a screenshot of 1.5.15

**Comment by daniel-toman** [comment]
I went to download 1.5.15, and I happened to notice that the Macs Fan Control app now installed in my Applications folder doesn't have the correct icon. (see screenshot) So that made me think something may have gone wrong with the auto update installation of this version. 

So I downloaded the latest version 1.5.16 again from the Releases page you linked. This does have the correct icon, and when I launch this version it looks like a Mac app as it should. Problem solved, I guess!

![Screen Shot 2024-09-04 at 2 41 20 PM](https://github.com/user-attachments/assets/21d2c39f-aebe-402f-b021-336b4b5b9fa9)
![Screen Shot 2024-09-04 at 2 42 54 PM](https://github.com/user-attachments/assets/11aec5a6-233f-4947-9314-ae1263ab45cf)
